pipeline-module/README.md

```diff
@@ -1,3 +1,131 @@
 # Pipeline module document template
 
-This is the template for SPHEREx pipeline module document repositories, built around `spherex-tex` and the `spherex` document class.
+This is the template for SPHEREx pipeline module document repositories (`MS` document category).
+These module descriptions are built around [`spherex-tex` and the `spherex` document class](https://github.com/SPHEREx/spherex-tex).
+
+## How to create a new document
+
+### 0. Prerequisites
+
+On your computer, ensure that [Cookiecutter](https://cookiecutter.readthedocs.io/en/latest/) is installed:
+
+```shell
+pip install -U cookiecutter
+```
+
+You also need Git >=
```

Comment:
Is there a version number missing from this line?

Reply:
Ah, I was trying to find when the -M flag was added to git, but could never find it. I'll change the wording on this.